Xamarin.Essentials/FileSystem/FileSystem.uwp.cs

```diff
@@ -24,6 +24,22 @@ static Task<Stream> PlatformOpenAppPackageFileAsync(string filename)
 
         internal static string NormalizePath(string path)
             => path.Replace('/', Path.DirectorySeparatorChar);
+
+        static string[] PlatformGetAppResourceDirectories(string path)
+        {
+            if (Directory.Exists(path))
```

Comment:
Should this also be normalized?

Reply:
Also, what happens if I pass a rooted path or change the cwd? Should we be checking for the location first?